driver/test_kubeflow_workloads.py

```diff
@@ -59,28 +60,62 @@
 
 KFP_PODDEFAULT_NAME = "access-ml-pipeline"
 
+CHARMS = {
```

Comment:
Cant we get this json from bundle-kubeflow repository https://github.com/canonical/bundle-kubeflow/tree/main/releases/1.9/stable instead ? By having it here we may have problems in the future when we decide to do some changes in the bundle file (we might forget to update this one).